server.js

```diff
@@ -0,0 +1,102 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMethod = request.method;
+  const reqVersion = request.httpVersion;
+
```

Comment:
The function declaration should not be implemented inside the connection callback. Currently your implementation recreates this function every time a request is received.

You will need to pass in the `response` object since you no longer be able to use scope to reference it.